voronoi.c

```diff
@@ -45,7 +62,7 @@ void generate_voronoi_seeds(Seed_t* seeds)
     for(int i = 0; i < SEED_NUM; i++) {
         seeds[i].loc.x = 1 + (rand() % (GRID_HEIGHT - 1));
         seeds[i].loc.y = 1 + (rand() % (GRID_WIDTH - 1));
-        seeds[i].tile = tiles[i % TileCount]->ascii;
+        seeds[i].tile = tiles[i % (TileCount-1)]->ascii;
```

Comment:
I told you I didn't like this solution, there should be a separate thing that knows the array size

Reply:
we will get there when we get there....